README.md

```diff
@@ -0,0 +1,16 @@
+# Image Captioning Library - Ovative Group
+## Omar Elsegeiny, Mason Myles, Ian King, Bohdan Ivanyshyn, Emily Anderson
+
+Our capstone project, a Python pipeline CLI library, offers a streamlined solution for generating captions from images. By simply providing URL(s) or a directory containing images, our library swiftly delivers insightful captions. Embedded within are pre-trained models for image captioning and optical character recognition, seamlessly integrated with a language generation model (LLM) for cohesive output. Tailored to meet the requirements of our client, Ovative Group, this tool serves as an in-house asset for efficiently captioning advertisements, offering both conciseness and detail in its captions.
```

Comment:
Say that the CLI provides a solution for "generating captions and a general overview for images." That will be a little more accurate since we have the OCR and LLM. 

I wouldn't say that the project delivers outputs swiftly since it's device dependent. Just remove that word.